.github/workflows/actions/publish-crate-package/action.yml

```diff
@@ -36,18 +39,18 @@ runs:
   - name: Cargo publish dry run
     shell: bash
     run: |
-      echo "Cargo publish '${{ inputs.package}}' package (dry run)"
-      cargo publish -p ${{ inputs.package}} --dry-run --no-verify
+      echo "Cargo publish '${{ inputs.package }}' package (dry run)"
+      cargo publish -p ${{ inputs.package }} --dry-run --no-verify ${{ inputs.publish_args }} 
 
   - name: Cargo package list
     shell: bash
     run: |
-      echo "Cargo package list '${{ inputs.package}}' package"
-      cargo package -p ${{ inputs.package}} --list
+      echo "Cargo package list '${{ inputs.package }}' package"
+      cargo package -p ${{ inputs.package }} --list
 
   - name: Cargo publish
     if: inputs.dry_run == 'false' && steps.check_version.outputs.should_deploy == 'true'
     shell: bash
     run: |
-      echo "Cargo publish '${{ inputs.package}}' package"
-      cargo publish -p ${{ inputs.package}} --token ${{ inputs.api_token }} --no-verify
+      echo "Cargo publish '${{ inputs.package }}' package"
+      cargo publish -p ${{ inputs.package }} --token ${{ inputs.api_token }} --no-verify ${{ inputs.publish_args }}
```

Comment:
To check if this is needed during publication?

Reply:
The `--features` option of the `cargo publish` is linked to the underlying `cargo build` (which we skip here with the `--no-verify` option), so no need to add the set of features here. 

Still, we can keep these `publish_args` in the action :+1: